layer.yaml

```diff
@@ -15,6 +15,11 @@ options:
     puppet-version: '3'
     silent: True
 defines:
+  install_java:
+    type: string
+    default: 'openjdk-8-jdk'
```

Comment:
I was basing my question off of the example value from [site.yaml](https://github.com/apache/bigtop/blob/master/bigtop-deploy/puppet/hieradata/site.yaml#L43) and the fact that the JRE would be a smaller download.  But it's not a big difference either way, as just curious if there was a justification other than "we might need something."